CONTRIBUTING.md

```diff
@@ -1,49 +1,52 @@
 # CONTRIBUTION PROCESS
-To contribute to this repository, please create a branch on your local 
+To contribute to this repository, please create a branch on your local
 clone, push that branch to the repository, and create a Pull Request (PR)
-for that branch. No commits can be directly added onto `master` - all 
+for that branch. No commits can be directly added onto `master` - all
 must come via a Pull Request.
 
 ### Reviewers
 All PRs must be approved by at least 2 reviewers before they can be merged.
 Any team members can act as reviewers, but it is suggested that in general
 each PR should have one reviewer from the 'software development' angle, and
-one reviewer from the 'data science' angle. All team members (dependent on 
+one reviewer from the 'data science' angle. All team members (dependent on
 their notifications settings) will be notified of new PRs, and are encouraged
 to provide additional review comments whenever they can.
 
 ### Work items
 Please link each PR to a work item from the current sprint. If no work items
-are linked, the PR cannot be merged - a new work item should be raised in 
+are linked, the PR cannot be merged - a new work item should be raised in
 order to facilitate capturing the work being done.
 
-If you reference a work item by id in a commit message - e.g. #12345 
-Fixed issue where data was eaten by a grue - DevOps will auto link the item 
+If you reference a work item by id in a commit message - e.g. #12345
+Fixed issue where data was eaten by a grue - DevOps will auto link the item
 to the commit, and therefore the PR.
 
-Otherwise you can manually link items at the point of PR creation, or 
+Otherwise you can manually link items at the point of PR creation, or
 afterwards, if necessary.
+#DS- Should there be more details added to the branch naming scheme other than the sprint work item number?
```

Comment:
The proposal is <ticket_id>-<brief description>, i.e. 29973-docker-configuration.